frontend/src/components/employees/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Table from '@material-ui/core/Table';
+import TableHead from '@material-ui/core/TableHead';
+import TableBody from '@material-ui/core/TableBody';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import toastr from 'toastr';
+import { StaffData, fetchStaff, selectStaff } from '../../features/staffSlice';
+import 'toastr/build/toastr.min.css';
+import employeesStyle from './employees.module.scss';
+
+const STAFF_FETCH_URL = 'http://localhost:8080/staff/get';
```

Comment:
I think we should move this to staffSlice. What do you think?

Reply:
We have a 'constants' directory, so I'd probably move it there.